src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/resource/ActivitiesResourceIntTest.kt

```diff
@@ -154,4 +166,113 @@ class ActivitiesResourceIntTest : IntegrationTestBase() {
       return response!!.courseActivityId
     }
   }
+
+  @Nested
+  inner class CreateOffenderProgramProfile {
+
+    lateinit var courseActivity: CourseActivity
+
+    private val createOffenderProgramProfileRequest: () -> CreateOffenderProgramProfileRequest = {
+      CreateOffenderProgramProfileRequest(
+        courseActivityId = courseActivity.courseActivityId,
+        bookingId = repository.lookupOffender("A1234TT")?.latestBooking()?.bookingId!!,
+        startDate = LocalDate.parse("2022-10-31"),
+        endDate = LocalDate.parse("2022-11-30"),
+      )
+    }
+
+    @BeforeEach
+    internal fun setup() {
+
```

Comment:
I think you need a tear down or other tests in the future might be affected

Reply:
main class tear down covers it